bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -43,10 +43,16 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: Instrumentation
+    - description: 'Instrumentation is the spec for OpenTelemetry instrumentation.
+        nolint: maligned'
```

Comment:
This line shouldn't be part of the CSV :-)

Reply:
removed the `nolint` annotation. I had to reshuffle the struct. It looks weird but there is no functional difference. Perhaps we can disable the `maligned` globally.